src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
The class name 'Square' is appropriate as it represents a specific shape, but ensure that the class does not contain any unnecessary empty lines at the beginning of its implementation.